src/launcher/java/org/truffleruby/launcher/options/CommandLineOptions.java

```diff
@@ -82,7 +82,8 @@ public void setArguments(String[] arguments) {
         return unknownArguments;
     }
 
-    public Map<String, String> getJVMOptions() {
+    // JVM options in polyglot format
```

Comment:
I would make this is a JavaDoc comment so it's shown nicely in the IDE.

Reply:
👍